packages/console/src/containers/AppContent/components/Topbar/index.tsx

```diff
@@ -23,6 +26,8 @@ function Topbar({ className }: Props) {
   return (
     <div className={classNames(styles.topbar, className)}>
       <LogtoLogo className={styles.logo} />
+      {isTenantDropdownReady && <TopbarDivider />}
```

Comment:
Could use a CSS pseudo-element?  So you could omit this separate divider element.

Reply:
Put this divider along with the dropdown component.

Reply:
Still not a good solution:
1. Do we need a different bg color in dark mode for this divider
2. Divider element should not be embedded in the component itself. Not flexible enough. What if the order of the Topbar nav changed?
3. Could draw this divider using CSS without introducing an extra SVG file. Looks like just a solid color bar.

Reply:
Updated.